cmake/GenerateHelptags.cmake.in

```diff
@@ -1,10 +1,10 @@
 if(DEFINED ENV{DESTDIR})
   file(TO_CMAKE_PATH
-    "$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc"
+    "$ENV{DESTDIR}/@CMAKE_INSTALL_FULL_DATADIR@/nvim/runtime/doc"
```

Comment:
Well, if you want I suppose it might be done that way, but I don't personally know how to do it.

My opinion is that using @BLAH@ substitution is better because it is very clear that it is going to be expanded upon by a configure script. Then the reader knows to look for the substitution declaration. If you just pass the variable in, it is more confusing since the variable name coincides with a standard cmake variable.